app/subscriber/src/components/tool-bar/ContentActionBar.tsx

```diff
@@ -33,13 +34,56 @@ export const ContentActionBar: React.FC<IContentActionBarProps> = ({
   removeFolderItem,
 }) => {
   const navigate = useNavigate();
+  const [{ frontPageImagesMediaTypeId, settings, isReady }] = useLookup();
+  const [isFrontPageImage, setIsFrontPageImage] = useState(true);
+  const [contentId, setContentId] = useState(content?.[0]?.id ?? null);
+
+  // Only show action items in the bar if it's NOT front page image content
+  const showActionsItems = !isFrontPageImage;
+
+  React.useEffect(() => {
+    // Every time content changes, assume it's a front page image until we know it's not
+    // this prevents rendering the action items until we know they're needed
+    if (contentId !== content?.[0]?.id) {
+      setIsFrontPageImage(true);
+      setContentId(content?.[0]?.id);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why is this needed?  What will happen if you include `contentId`?

Reply:
Oops, great catch. 'twas needed before but not anymore. Thanks!